zedUpload/datastore_http_test.go

```diff
@@ -268,29 +271,101 @@ func testHTTPDatastoreNegative(t *testing.T) {
 	})
 }
 
+// testHTTPDatastoreRepeat tests that the client can handle lost packets
+// during a large download stream and recover, by using the Range HTTP header.
+// The tests sets up an "unstable" proxy, which is configured to start dropping and
+// delaying packets after a certain number of bytes have been transferred.
+//
+// The test works as follows:
+//
+//		1- create a large input file with random data, e.g. 100MB
+//		2- hash the input file
+//		3- create a test http server configured to serve the input file
+//		4- start the unstable proxy, pointing to the test http server as the origin,
+//		   configured to drop packets for a fixed period of time, e.g. 40 seconds, after half the file has been transferred
+//		5- download the file using the unstable proxy, which should download half the file, fail for the fixed period of time,
+//	    and then successfully download the rest
+//		6- when download completes, hash the downloaded file
+//		7- check that the hashes match
 func testHTTPDatastoreRepeat(t *testing.T) {
 	if testing.Short() {
 		t.Skip("skipping HTTP repeat test suite.")
 	} else {
```

Comment:
```suggestion
		t.Skip("skipping HTTP repeat test suite.")
		return
	} 
```

Reply:
No need. `t.Skip()` is equal to `t.Log()` followed by `t.SkipNow()`.

Reply:
Sure, as long as you remove the `else` part